core/safe_refcount.cpp

```diff
@@ -5,8 +5,8 @@
 /*                           GODOT ENGINE                                */
 /*                      https://godotengine.org                          */
 /*************************************************************************/
-/* Copyright (c) 2007-2018 Juan Linietsky, Ariel Manzur.                 */
-/* Copyright (c) 2014-2018 Godot Engine contributors (cf. AUTHORS.md)    */
+/* Copyright (c) 2007-2017 Juan Linietsky, Ariel Manzur.                 */
```

Comment:
This would need to be changed back to 2018, but why add this commit in the first place? Is it necessary for arm64 support? I'm not so keen on changing such low level code in the 2.1 branch (cc @hpvb).